File: Programmers/Level1/k번째수.py
def solution(array, commands):
    answer = []
    for command in commands:
        i, j, k = command
        temp = []
        for i in range(i-1, j, 1):
            temp.append(array[i])
        temp.sort()
        answer.append(temp[k-1])
    return answer

array = [1, 5, 2, 6, 3, 7, 4]
commands = 	[[2, 5, 3], [4, 4, 1], [1, 7, 3]]
print(solution(array, commands))

File: Programmers/Level1/k번째수/k번째수.js
function solution(array, commands) {
    const answer = [];
    for (let k=0; k<commands.length; k++) {
        const [l1, l2, l3] = commands[k];
        let myArr = array.slice(l1-1, l2);
        for (let i=0; i< myArr.length; i++) {
            for (let j=i; j<myArr.length; j++) {
                if (myArr[i] > myArr[j]) {
                    var num = myArr[i];
                    myArr[i] = myArr[j];
                    myArr[j] = num;
                }
            }
        }
        answer.push(myArr[l3-1]);
    }
    return answer
}

const array = [1, 5, 2, 6, 3, 7, 4];
const commands = [[2, 5, 3], [4, 4, 1], [1, 7, 3]];

console.log(solution(array, commands));